src/main/java/com/hivemq/plugin/mqttmessage/config/MqttMessageLogConfigReader.java

```diff
@@ -0,0 +1,51 @@
+package com.hivemq.plugin.mqttmessage.config;
+
+import com.hivemq.spi.config.SystemInformation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class MqttMessageLogConfigReader {
+
+    private static final Logger log = LoggerFactory.getLogger(MqttMessageLogConfigReader.class);
+
+    private final Properties properties = new Properties();
+    private final SystemInformation systemInformation;
+
+    private static final String PROPERTIES_FILE_NAME = "mqttMessageLog.properties";
+
+    @Inject
+    public MqttMessageLogConfigReader(SystemInformation systemInformation) {
+        this.systemInformation = systemInformation;
+    }
+
+    @PostConstruct
+    public void postConstruct() {
+        final File configFolder = systemInformation.getConfigFolder();
+        final File pluginFile = new File(configFolder, PROPERTIES_FILE_NAME);
+
+        log.info("will try to read config properties from {}", PROPERTIES_FILE_NAME);
```

Comment:
this should be in debug log. 

please add the name of the plugin to the log statement.